.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Report a bug in kf
+title: ''
+labels: bug
+assignees: ''
+
+---
+<!-- If you need to report a security issue with kf, send an email to kf-security@google.com. -->
+
+## What version of kf?
```

Comment:
Can we add commands to answer these questions?
```kf version```

Reply:
We do have a kf version command already.

There's another issue pending to add a larger printout we can use when asking people about their environment to provide support #318

Reply:
Sorry, I meant can we add instructions to run these commands? That way we can expand them later, but the issue template stays the same